musicbot/block12/bot.py

```diff
@@ -67,9 +74,10 @@ async def cmd_billboard(self, author, channel, guild, leftover_args):
         
     @overrides(MusicBot)
     def _init_player(self, player, *args, guild = None):
-        super()._init_player(player, *args, guild = guild)
-        BillboardManager.addEventEmitter(player)
-        return player
+        Block12MusicPlayerInjector(player)
+        player_to_return = super()._init_player(player, *args, guild = guild)
```

Comment:
unneeded variable (player_to_return)?

Reply:
some calls for `_init_player` also returns the player so this function also needs to return the player